src/OrchardCore.Modules/OrchardCore.Deployment/Views/DeploymentPlan/Display.cshtml

```diff
@@ -112,21 +116,30 @@
     </div>
 </div>
 <script at="Foot" type="text/javascript">
+    function replaceDiacritics(str) {
```

Comment:
Is this for a specific locale?!!

Reply:
In French for example, to search some words with accents.

Reply:
IMHO this should be locale independent